source/includes/steps-install-shell-macos.yaml

```
@@ -9,10 +9,26 @@ source:
   ref: download-archive
 ref: download-archive-macos
 ---
-source:
-  file: steps-install-shell-base.yaml
-  ref: extract-archive
-ref: extract-archive-macos
+title: "Extract the files from the downloaded archive."
+ref: extract-zip-macos
+level: 4
+content: |
+
+   Run the following command from the directory containing the
+   ``mongosh`` ``.zip`` archive:
+   
+   .. code-block:: sh
+
+      unzip mongosh-{version}-darwin-x64.zip
+
+   If your web browser automatically extracts the archive as
+   part of the download or you extract the archive without the
+   ``unzip`` command, you may need to run the following command to
+   make ``mongosh`` executable:
+
+   .. code-block:: sh
+
+      chmod +x /bin/mongosh
```

```suggestion
      chmod +x bin/mongosh
```

Unless this would actually be installed in /bin, which would be fairly untypical?

---

Updated and tweaked this a bit to clarify where you need to run this command. Thanks for flagging this.